stock_prices/stock_prices.py

```diff
@@ -3,7 +3,17 @@
 import argparse
 
 def find_max_profit(prices):
```

Comment:
The time complexity of your solution for the stock prices problems runs at O(n^2). Here is my solution that has time complexity of O(n) which is better than O(n^2):

```
def find_max_profit(stocks):
  min = 0
  max = None

  next_stock = 1

  while next_stock != len(stocks):
    val = stocks[next_stock] - stocks[min]

    if stocks[min] > stocks[next_stock]:
      min += 1

    if max == None or val > max:
      max = val

    next_stock += 1

  return max
```